### PR TITLE
Arm64 Support for Kafka

### PR DESCRIPTION
**Pull request by stephenkaye-ho**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -125,15 +123,14 @@ services:
     <<: *certificate-creation
     command: ["kafka-consumer"]
 
+  # Command to read topic
+  # kafka-console-consumer --bootstrap-server kafka:9093 --from-beginning --topic callisto-timecard-timeentries --consumer.config /tmp/kafka-consumer.properties
```
Comment (taomoh) [inline docker-compose.yml]: Remove commented out code

File: docker-compose.yml [hunk 2]
```diff
@@ -58,60 +58,58 @@ services:
...
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:SSL,SSL:SSL
+      KAFKA_LISTENERS: CONTROLLER://kafka:9092,SSL://kafka:9093
+      KAFKA_LOG_DIRS: '/tmp/kraft-combined-logs'
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_PROCESS_ROLES: broker,controller
+      KAFKA_SSL_CLIENT_AUTH: required
+      KAFKA_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: " "
+      KAFKA_SSL_KEY_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_KEYSTORE_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_KEYSTORE_FILENAME: keystore/kafka/kafka.keystore.jks
```
Comment (taomoh) [inline docker-compose.yml:88]: These are relative paths. Is this intended?

File: docker-compose.yml [hunk 3]
```diff
@@ -58,60 +58,58 @@ services:
...
+      KAFKA_SSL_PRINCIPAL_MAPPING_RULES: RULE:^C=GB,O=UK Home Office,CN=Callisto (.+)$$/$$1/,DEFAULT
+      KAFKA_SSL_TRUSTSTORE_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_TRUSTSTORE_FILENAME: keystore/kafka/kafka.truststore.jks
```
Comment (taomoh) [inline docker-compose.yml:91]: These are relative paths. Is this intended?

File: kafka/setup_scripts/topics.txt [hunk 4]
```diff
@@ -1 +1,2 @@
-timecard-timeentries
\ No newline at end of file
+timecard-timeentries
```
Comment (taomoh) [inline kafka/setup_scripts/topics.txt]: can the old topic name be removed now?

File: docker-compose.yml (hunk 2 above)
Comment (stephenkaye-ho) [reply docker-compose.yml:88]: Yes, the setup in this image appends these paths to a predefined location which is where the keystore volume is mounted for this service
```/etc/kafka/secrets```
https://github.com/UKHomeOffice/callisto-localdev/pull/15/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R65

File: docker-compose.yml (hunk 3 above)
Comment (stephenkaye-ho) [reply docker-compose.yml:91]: > Yes, the setup in this image appends these paths to a predefined location which is where the keystore volume is mounted for this service `/etc/kafka/secrets` https://github.com/UKHomeOffice/callisto-localdev/pull/15/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R65

File: docker-compose.yml (hunk 1 above)
Comment (stephenkaye-ho) [reply docker-compose.yml]: Moved to README

File: kafka/setup_scripts/topics.txt (hunk 4 above)
Comment (stephenkaye-ho) [reply kafka/setup_scripts/topics.txt]: I've removed and taken out the related permissions as well

**Review by taomoh — APPROVED** [review]

**Review by sewerynmi — APPROVED** [review]

**Pull request merged**